source/documentation/virtual_machine_management_guide/topics/Affinity_groups.adoc

```diff
@@ -2,13 +2,13 @@
 [id="affinity-groups"]
 = Affinity Groups
 
-Affinity groups help you determine where selected virtual machines run in relation to each other and specified hosts. This capability helps manage workload scenarios such as licensing requirements, high-availability workloads, and disaster recovery.
+You can create *Affinity Groups* help determine where selected virtual machines run in relation to each other and to specified hosts. This capability helps manage workload scenarios such as licensing requirements, high-availability workloads, and disaster recovery.
 
 .The VM Affinity Rule
-When you create an affinity group, you select the virtual machines that belong to the group. To define _where these virtual machines can run in relation to each other_, you enable a *VM Affinity Rule*: A positive rule tries to run the virtual machines together on a single host; a negative affinity rule tries to run the virtual machines apart on separate hosts. If the rule cannot be fulfilled, the outcome depends on whether the weight or filter module is enabled.
```

Comment:
"where" suggest a physical location. Use "when" if you want to refer to time; "which virtual machines" if you want the user to choose which VMs are involved.

Reply:
Yes, it is a physical location that is being discussed - this is as form of load distribution and  risk management ( trying to make sure everything is not onb one VM that may fail and bring down the whole business).